shared/chat/conversation/messages/wrapper/send-indicator/index.js

```diff
@@ -102,14 +99,22 @@ class SendIndicator extends React.Component<Props, State> {
       return null
     }
     return (
-      <SendIcon
-        status={this.state.iconStatus}
-        style={{...this.props.style, opacity: this.state.visible ? 1 : 0}}
+      <Kb.Icon
```

Comment:
on iOS this wraps to the next line 
![send-wrapping](https://user-images.githubusercontent.com/11968340/49451467-3d9e7d80-f7ad-11e8-9665-68967ec35c34.gif)